packages/abstract-provider/src.ts/index.ts

```diff
@@ -34,27 +34,28 @@ export type HederaTransactionRecord = {
     chainId: number,
     transactionId: string,
     result: string,
-    amount: number,
-    call_result: string,
-    contract_id: string,
-    created_contract_ids: string[],
-    error_message: string,
-    from: string,
-    function_parameters: string,
-    gas_limit: number,
-    gas_used: number,
-    timestamp: string,
-    to: string,
-    block_hash: string,
-    block_number: number,
-    hash: string,
-    logs: {}
+    amount?: number,
+    call_result?: string,
+    contract_id?: string,
+    created_contract_ids?: string[],
+    error_message?: string,
+    from?: string,
+    function_parameters?: string,
+    gas_limit?: number,
+    gas_used?: number,
+    timestamp?: string,
+    to?: string,
+    block_hash?: string,
+    block_number?: number,
+    hash?: string,
+    logs?: {},
+    accountAddress?: string
 }
   
 export interface TransactionResponse extends Transaction {
     hash: string;
     timestamp?: string,
-    from: string;
+    from?: string;
```

Comment:
Why did we make the `from` optional?
I think every transaction should `always` have a sender. @Daniel-K-Ivanov  am i missing something?

Reply:
My mistake. At the time I thought that `from` couldn't be populated. It is not optional any more.